rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
> > Specs for files in the lib of a rails project would, optimally, go into spec/lib/.

> Are you sure? That's the first time I'm hearing about this. The RSpec 
> docs (search for "Tips on Spec Location") suggest treating app and lib 
> the same in this respect.

Fwiw, i have always considered it a bug in rspec-mode that in a rails app it put specs for `lib/` directly in the spec dir. It seems incongruous to choose to segregate the source files and then to mix the spec files all willy-nilly.  I suppose the correctness of the current behavior might be debatable but i dislike it.

If there is no consensus on where spec files for `lib/`of a rails app should go (or if the consensus doesn't match my personal opinion :smile:) perhaps a customization variable like `rspec_primary_source_dirs` is a reasonable compromise.